packages/moonstone/Popup/Popup.js

```diff
@@ -325,7 +324,7 @@ class Popup extends React.Component {
 		scrimType: PropTypes.oneOf(['transparent', 'translucent', 'none']),
 
 		/**
-		 * When `true`, the popup includes a close button; when `false`, none is included.
```

Comment:
Separate short description of `scrimType` from additional information with a blank line.